code_to_review.py

```diff
@@ -0,0 +1,343 @@
+import json
```

Comment:
## general suggestions: 

1. Consider breaking into smaller functions.
2. Create be multiples lambda and make use of step-functions to improve this lambda and reduce its vulnerability in case one day we have huge file to process.
3. use config or env variable instead of hardcoded values
4. Make proper logging
5. Add validation of function inputs
6. Make naming clearer and consistent
7. Add a unique coding style issues and stick with it. chose between single quote and double quote.
5. Add timeouts on requests and async code
6. Add functions comments, type hinting, [pydantic](https://github.com/pydantic/pydantic) can help us achieve this.
7. Avoid repetition.